list-your-project/other-dashboards/fees-adapters.md

```diff
@@ -0,0 +1,308 @@
+# Building Fees Adapters
+
+This guide will help you create adapters for the [fees dashboard](https://defillama.com/fees) that track fees and revenue metrics for DeFi protocols.
+
+## Adapter Versions
+
+Before creating a fees adapter, you need to decide which version to use:
+
+- **V1 Adapters**: Run for fixed time periods (00:00 to 23:59 of a given day)
+- **V2 Adapters**: Accept arbitrary start and end timestamps as input
+
+The key differences:
+
+- V1 adapters are suitable for APIs that only return daily data with no way to specify custom time ranges
+- V2 adapters allow for more flexible time ranges and real-time data (e.g., pulling data from 2 PM yesterday to 2 PM today)
+- Use the `runAtCurrTime` flag for adapters that can only return the most recent 24-hour data with no historical support
+
+## Implementation Steps
+
+### 1. Identify Supported Chains
+
+Identify all chains your protocol operates on by referencing the [chains.ts](https://github.com/DefiLlama/dimension-adapters/blob/master/helpers/chains.ts) file.
+
+### 2. Check for Existing Adapters
+
+Before implementation, search DeFiLlama and the dimensions repo to ensure an adapter doesn't already exist for your project.
+
+### 3. Define Start Dates
+
+For each chain, determine the deployment date of your protocol in YYYY-MM-DD format or as a unix timestamp. This helps with data backfilling.
+
+### 4. Implement Fetch Function
+
+The core of your adapter is the `fetch` function that collects fees data. You'll need to implement logic to return some or all of these dimensions:
+
+- **dailyFees**: All fees collected from all sources (including staking rewards, yields, etc.)
```

Comment:
id just do dailyFees, dailayRevenue, dailyHoldersRevenue
and then a note saying that if the protocol implements a token system with bribes, to add dailyBribes